src/app/core/export-config.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+
+@Injectable()
+/**
+ * Admin management of users
+ */
+export class ExportConfigService {
+
+	constructor(private http: HttpClient) {}
+
+	postExportConfig(type: string, config: any): Observable<any> {
+		return this.http.post(
+			'/requestExport',
```

Comment:
@jrassa should this be `/api/requestExport` ?

Reply:
good catch